frontend/package.json

```diff
@@ -17,7 +17,7 @@
     "cy:version": "cypress version",
     "cy:open": "cypress open",
     "cy:run": "cypress run",
-    "compile-nextjs-config": "tsc --project nextjs.tsconfig.json"
+    "compile-nextjs-config": "ncc build next.config.ts --transpile-only --tsconfig-path nextjs.tsconfig.json --out . && mv index.js next.config.js"
```

Comment:
It happens to be the fastest-to-implement way to compile `next.config.ts` at the moment.